### ISSUE TITLE
currentPage works just like initialPage?

### ISSUE THREAD

**Issue by njnest** [phil-r/react-native-looped-carousel#127]
Hi. When I change the currentPage nothing changes. 
Is it possible to manage a page change outside the carousel?

**Comment by viktorsec** [comment]
Yep, that's exactly what it is for! We should add it to the docs.
